pkgs/development/python-modules/mkdocs-redoc-tag/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, beautifulsoup4
+, buildPythonPackage
+, fetchFromGitHub
+, mkdocs
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-redoc-tag";
+  version = "0.1.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`.

---

I implemented your suggestion. Thanks!